api/c/indigo/src/indigo_molecule.cpp

```diff
@@ -508,12 +510,14 @@ CEXPORT int indigoLoadQueryMolecule(int source)
 
         loader.stereochemistry_options = self.stereochemistry_options;
         loader.treat_x_as_pseudoatom = self.treat_x_as_pseudoatom;
+        loader.dearomatize_on_load = self.dearomatize_on_load;
+        loader.arom_options = self.arom_options;
 
         std::unique_ptr<IndigoQueryMolecule> molptr = std::make_unique<IndigoQueryMolecule>();
 
         QueryMolecule& qmol = molptr->qmol;
 
-        loader.loadQueryMolecule(qmol);
+        loader.loadMolecule(qmol);
```

Comment:
?

Reply:
loadMolecule now can load both: queries and molecules. Actually we don't need separate methods to do that.